### ISSUE TITLE
The State has an Issue in RU/UA Wording + How to Disperse the Angry Crowd

### ISSUE THREAD

**Issue by ghost** [MachineGunnur/GOG-Games#420]
> Most of these new games were purchased and uploaded by a P2P group called R.G. GOGFAN. 

**Finally, confessed.** Took you long enough, years. Good.

> due to the current on-going conflict between Ukraine and Russia 

This is the whole reason that forced me to respond. **This part is going to make many sides angry! Cut it!** Russia's gov't prohibits the word "war," allows something like "conflict," or "special military operation." So you will make Russia's opposition and Ukranians think you are pro-war or deceived (I sort of do think so now), and a skilled foreigner might read it this way too, you know. If do replace by "war," you'll piss off pro-gov't folks or who believes in that it isn't a war at all. Both ways you lose someone.

I am a native speaker, I know that all too well, so don't play with fire, also delete my comment and delete own edit history. Simple **"as GOG.com has decided to suspend doing business with Russia" is enough.** You might generate dramas with others  or lose followers by being so careless on wording.

> Unfortunately, I do not have the financial means to purchase every single game that is released on GOG.com.

**You don't have "any" means, in practice, you said it above yourself.** The vast majority of games are brought from others, except a few, all new are practically from others. **The latest ones were from the spot we both know,** and are brought by others.

Don't be so afraid, nobody will abandon you for what you are, these people will pay for your hosting, just always be truthful with the hand that feeds you, and don't deny a discussion like you tend to sadly rudely do. Everyone respects the truth, unlike a lie: Lies and ingorance is what destroys relationships. 

> I have already received very rude emails from individuals who assume they are entitled to every new game GOG puts out. This is very disheartening.

Disheartening? **These people believe in the story that you buy games!** Be clear, say you copy/steal it from other more "productive" pirates and don't buy, and get peace, very easy to do. If I were you, that would be the 1st thing to do to calm down.

Sure these people are way too impatient, no excuses, but you should've not lied to them all. **Tell the truth,** tell to bother a site that actually buys games and that you use, everyone wins then: You get no angry people, the people get the demands fullfilled, and you get new games. The other sites are too often inconvenient to use unlike your site, so you shouldn't lose supporters, we can accept the truth just fine.

In my view both sides of the argument are misunderstanding each other.

> The bills for next month are also due on July 14th. If this is not paid, I will have to severely cut costs (such as remove torrents as I was paying out of pocket myself for seed boxes each month - 30 euros). 

**Alright, cut.** I always viewed P2P as a gimmick. Instead, if success happens, maybe do us a favour and do buy games for €30 per month or so. Yes it's small, but it's better than nothing and we are in a bad situation on the incoming queue. New games are better than silly P2P. Everyone will love you actually doing something great and actually buying games using the funds, I guarantee.

In short, be truthful, be careful, and have a nice day.

**Comment by MachineGunnur** [comment]
oh look you. go away.
